### DISCUSSION TITLE
Aquifer infiltration parameter

### DISCUSSION THREAD

**Discussion by davidezocca** [CH-Earth/summa#476] [Q&A] [answered]
Hi! I'm trying to set up SUMMA, and I had a question about the parameter that regulates the infiltration from the soil into the groundwater. Is it possible to set a different value for each HRU?
Also, in mizoroute do you know how to set the input to a channel reach as continuous along its length?
Thanks!

**Comment by wknoben** [comment, marked as answer]
You can use the `trial parameters` file to specify parameter values at the HRU-level. The SUMMA docs provide some more details:

- Main article: https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#attribute-and-parameter-files
- Trial parameter file: https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#infile_trial_parameters

As for the mizuRoute question you're probably better off asking there. Tentatively I'd say that that isn't (currently) possible because mizuRoute assumes that any runoff from a basin enters the channel at the basin outlet (i.e. it will go into the channel segment that belongs to the basin but no within-channel routing in the basin is applied). You may be able to use either SUMMA's `subRouting   timeDlay` model decision or mizuRoute's `<doesBasinRoute> 1` control file option to activate (within-basin) hillslope routing. This will give you a travel time distribution for a basin's runoff before it enters the channel. Take care not to turn this option on in both SUMMA and mizuRoute together, or the hillslope routing will be applied twice.
